samples/drivers/i2s/echo/dts/bindings/wolfson,wm8731.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: WM8731 Audio CODEC
```

Comment:
Why isn't this in the top level dts/bindings directory?

Reply:
It seemed more appropriate to me, as such nodes are used only in this one sample and I'm not providing a driver for this CODEC, only a simple initialization routine. But I can move the file if you think it would be better to place it in the top level bindings.

Reply:
>> Why isn't this in the top level dts/bindings directory?
>
> It seemed more appropriate to me, as such nodes are used only in this one sample and I'm not providing a driver for this CODEC, only a simple initialization routine.

OK, that's what I figured. That seems reasonable, but could you please add a comment explaining that in the file itself?

Also cc @galak

Reply:
Added.